app/markets/[id]/MarketDetails.tsx

```diff
@@ -0,0 +1,97 @@
+"use client";
+
+import { OutcomeBar, Swapbox } from "@/app/components";
+import { useQuery } from "@tanstack/react-query";
+import { getMarket } from "@/queries/omen";
+import { Button, IconButton, Tag } from "swapr-ui";
+import { remainingTime } from "@/utils/dates";
+import Link from "next/link";
+
+interface MarketDetailsProps {
+  id: string;
+}
+
+export const MarketDetails = ({ id }: MarketDetailsProps) => {
+  const { data, error } = useQuery({
+    queryKey: ["getMarket", id],
+    queryFn: async () => getMarket({ id }),
+  });
+
+  if (error) throw error;
+  if (!data?.fixedProductMarketMaker) return <LoadingMarketDetails />;
+
+  const market = data.fixedProductMarketMaker;
+
+  const closingDate = new Date(+market.openingTimestamp * 1000);
+
+  return (
+    <div className="space-y-4">
+      <Link className="flex space-x-1.5 items-center w-fit" href="/">
```

Comment:
This section should do hover together

```suggestion
      <Link className="flex space-x-1.5 items-center w-fit group" href="/">
```

Reply:
then we need to add `group-hover:` modifier bellow.

https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state

But as they are imported components not sure if it will take to much work. This is a nice to have.